spec/rates_store/memory_spec.rb

```diff
@@ -60,3 +60,57 @@
     end
   end
 end
+
+# Historical
+describe Money::RatesStore::Memory do
```

Comment:
nit: a bit more idiomatic RSpec would be to use `context 'when using historical store' do` in the main `describe`

Reply:
This is a great nit, I'm not happy with how I organized it.